.github/workflows/release.yml

```diff
@@ -219,3 +249,51 @@ jobs:
         asset_path: ./pganalyze-collector_${{ steps.get_version.outputs.version }}_arm64.deb
         asset_name: pganalyze-collector_${{ steps.get_version.outputs.version }}_arm64.deb
         asset_content_type: application/octet-stream
+
+    - name: Upload helm package
```

Comment:
We could consider removing this step actually, because we will upload the package to s3. The downside of doing this is:
* the same package will be in GitHub release and s3
* the name will show up like pganalyze-collector-0.55.0.tgz in GitHub release, which might not be so obvious that it's about the helm chart